test/witan/models/dem/ccm/core/projection_loop_test.clj

```diff
@@ -74,8 +78,8 @@
           latest-yr (:loop-year births-added)
           latest-newborns (i/query-dataset popn-with-births {:year latest-yr :age 0})]
       (is (= 2 (first (:shape latest-newborns))))
```

Comment:
Core.matrix datasets have 3 attributes than can be accessed by treating the dataset as a map. One of this attribute is the shape of the dataset.

Let's take this example dataset:
```Clojure
(def dataset-test
    (clojure.core.matrix.dataset/dataset [{:a 1 :b 11}
                                          {:a 2 :b 12}
                                          {:a 3 :b 13}]))
```
```Clojure
dataset-test

| :a | :b |
|----+----|
|  1 | 11 |
|  2 | 12 |
|  3 | 13 |
```